src/Service.Host/client/src/components/__tests__/RunMrp.test.js

```diff
@@ -0,0 +1,271 @@
+            runLogIdCurrentlyInProgress: null,
+            links: [
+                { href: '/purchasing/material-requirements/last-run', rel: 'self' },
+                {
+                    href: '/purchasing/material-requirements/run-logs?jobRef=AAJADE',
+                    rel: 'last-run-log'
+                },
+                { href: '/purchasing/material-requirements/run-mrp', rel: 'run-mrp' }
+            ]
+        },
+        loading: false
+    },
+    runMrp: {},
+    mrpRunLog: {
+        item: {
+            mrRunLogId: 11662,
+            jobRef: 'AAJADE',
+            buildPlanName: 'MASTER',
+            runDate: '2022-05-04T09:36:39',
+            runDetails:
+                'rdf_name:MASTER\nrdf source:Remote MRP Run\ninput build plan:MASTER\noutput build plan:MASTER\ndo lrp:Y\ntest only:N\n',
+            fullRun: 'Y',
+            kill: null,
+            success: 'Y',
+            loadMessage:
+                ' 09:36:39:Starting \n 09:37:22:Loaded BOMS.\n 09:37:22:Starting to load sales forecast \nRead:33167, Written:33167 09:37:36:Loaded sales forecast.\n 09:37:36:Making build plan \n 09:48:18: Build plan made successfully. Jobref:AAJADE\n 09:48:18:Starting LRP ..\n 09:48:49:ST_LRP_PACK.LRP  Run successful\n 09:48:49:* RUN FINISHED SUCCESSFULLY *\n',
+            mrMessage:
+                'Updating mr for part WOOD 002Updating mr for part WOOD 003Updating mr for part WOOD 003/2Updating mr for part WOOD 004Updating mr for part WOOD 005Updating mr for part WOOD 006Updating mr for part WOOD 007Updating mr for part WOOD 008Updating mr for part WOOD 008/AUpdating mr for part WOOD 008/PUpdating mr for part WOOD 009Updating mr for part WOOD 010Updating mr for part WOOD 010/LUpdating mr for part WOOD 010/RUpdating mr for part WOOD 010/SUpdating mr for part WOOD 011Updating mr for part WOOD 011/BUpdating mr for part WOOD 011/SUpdating mr for part WOOD 012Updating mr for part WOOD 012/BUpdating mr for part WOOD 012/SUpdating mr for part WOOD 013Updating mr for part WOOD 013/SUpdating mr for part WOOD 014Updating mr for part WOOD 015Updating mr for part WORKTOPUpdating mr for part WPLATEUpdating mr for part WPLATE/PUpdating mr for part WPLINTHUpdating mr for part WPLINTH/2Updating mr for part WPLINTH/PUpdating mr for part WPLINTHBUpdating mr for part XELOS VR PLATEUpdating mr for part XOVER 001Updating mr for part YPLINTHUpdating mr for part lowercase partUpdating mr for part new98Updating mr for part newishset recommended purch order Finished successfully.',
+            dateTidied: null,
+            links: [{ href: '/purchasing/material-requirements/run-logs/11662', rel: 'self' }]
+        },
+        loading: false
+    }
+};
+const stateWithMrpRunResult = {
+    ...stateWithMasterNotRunning,
+    runMrp: {
+        data: {
+            success: true,
+            message: 'MRP requested with run log id 11663',
+            processHref: '/purchasing/material-requirements/run-logs/11663'
```

Comment:
Should have said this on the last PR, but i thought putting the processHref in the links array of the resource was more in line with all our hateoas stuff

Reply:
it's a very fair point. especially when you mentioned the possibility of a cancel href as per mailchimp